src/bridge/jsHelpers.ts

```diff
@@ -116,11 +122,19 @@ export const makeSync =
         const needClear = initial.id !== accountId;
 
         try {
+          const freshAddressPath = getSeedIdentifierDerivation(
+            initial.currency,
+            initial.derivationMode
+          );
+
           const shape = await getAccountShape(
             {
               currency: initial.currency,
               id: accountId,
+              index: initial.index,
```

Comment:
Is the index merely the last part of the derivation path?

Reply:
Soo actually I got bit by that, I just found out that for some obscure reason, the CLI provides a full derivation path (`44'/0'/0'/1/5`) but LLD provides only the root path! (`44'/0'`)
Pretty weird tbh. But I need to keep this `index` param to be safe...